docs/how-to/secure.md

```diff
@@ -53,12 +44,19 @@ docker cp ca-certs.pem pmm-server:/srv/nginx/ca-certs.pem
 docker cp dhparam.pem pmm-server:/srv/nginx/dhparam.pem
```

Comment:
I think in this case those also need to be owned by root. That can be resolved by going into the container and chown to root there - had exactly this case with the prospect recently (they had no root on the host).

Reply:
if @rasika-chivate can show me how to create content that is collapsed I'll happily go into more detail on how to accomplish this on the client side!  The page is getting pretty busy so being able to show the "docker" way start to finish and not have to also display the AMI/OFV way means 2-3x the content.

Reply:
We can add something like this:

<details>
  <summary>Copying Certificates</summary>


  If PMM Server is running as a Docker image, use `docker cp` to copy certificates. This example copies certificate files from the current working directory to a running PMM Server docker container.

```sh
docker cp certificate.crt pmm-server:/srv/nginx/certificate.crt
docker cp certificate.key pmm-server:/srv/nginx/certificate.key
docker cp ca-certs.pem pmm-server:/srv/nginx/ca-certs.pem
docker cp dhparam.pem pmm-server:/srv/nginx/dhparam.pem
```
</details>